scripts/base/packet-protocols/tcp/main.zeek

```diff
@@ -0,0 +1,5 @@
+module PacketAnalyzer::TCP;
+
+event zeek_init() &priority=20
```

Comment:
The `zeek_init` events in these files probably should be commented out if they're not doing anything. No use adding yet another event to be called that's just a no-op.